src/dispatch/plugins/dispatch_jira/plugin.py

```diff
@@ -465,7 +465,7 @@ def create_case_ticket(
                 "issuetype": issuetype,
                 "assignee": assignee,
                 "reporter": reporter,
-                "summary": title,
+                "summary": title.strip(),
```

Comment:
Since a new line could be in the middle of the summary, I think we'll need to use `.replace()`
```suggestion
                "summary": title.replace("\n", ""),
```